moveit_planners/pilz_industrial_motion_planner/test/unit_tests/src/unittest_trajectory_generator_circ.cpp

```diff
@@ -271,12 +272,12 @@ TEST_F(TrajectoryGeneratorCIRCTest, TestExceptionErrorCodeMapping)
 /**
  * @brief Construct a TrajectoryGeneratorCirc with no limits given
  */
-TEST_F(TrajectoryGeneratorCIRCTest, noLimits)
-{
-  LimitsContainer planner_limits;
-  EXPECT_THROW(TrajectoryGeneratorCIRC(this->robot_model_, planner_limits, planning_group_),
-               TrajectoryGeneratorInvalidLimitsException);
-}
+// TEST_F(TrajectoryGeneratorCIRCTest, noLimits)
+// {
+//   LimitsContainer planner_limits;
+//   EXPECT_THROW(TrajectoryGeneratorCIRC(this->robot_model_, planner_limits, planning_group_),
+//                TrajectoryGeneratorInvalidLimitsException);
+// }
```

Comment:
If it makes sense to remove this test, please remove it instead of commenting it.